File: SDKs/amazonaws.com/opsworks/2013-02-18/python/src/sdk/models/shared/describestacksresult.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class DescribeStacksResult:
    r"""DescribeStacksResult
    Contains the response to a <code>DescribeStacks</code> request.
    """
    
    stacks: Optional[List[Stack]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Stacks') }})
    

File: SDKs/amazonaws.com/opsworks/2013-02-18/typescript/src/sdk/models/shared/describestacksresult.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { Stack } from "./stack";



// DescribeStacksResult
/** 
 * Contains the response to a <code>DescribeStacks</code> request.
**/
export class DescribeStacksResult extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=Stacks", elemType: Stack })
  stacks?: Stack[];
}
